assets/js/modules/pagespeed-insights/components/dashboard/DashboardPageSpeedCTA.js

```diff
@@ -51,39 +45,32 @@ function DashboardPageSpeedCTA() {
 		return null;
 	}
 
-	const handleSetUpClick = async () => {
-		try {
-			await activateOrDeactivateModule( data, 'pagespeed-insights', true );
-			global.location = getReAuthURL( 'pagespeed-insights' );
-		} catch ( err ) {
-			showErrorNotification( GenericError, {
-				id: 'pagespeed-insights-setup-error',
-				title: __( 'Internal Server Error', 'google-site-kit' ),
-				description: err.message,
-				format: 'small',
-				type: 'win-error',
-			} );
-		}
-	};
+	const description = __( 'Google PageSpeed Insights gives you metrics about performance, accessibility, SEO and PWA.', 'google-site-kit' );
 
 	return (
 		<div className="
 			mdc-layout-grid__cell
 			mdc-layout-grid__cell--span-12
 		">
-			<CTA
-				title={ ! connected && active
-					? __( 'Complete PageSpeed Insights activation.', 'google-site-kit' )
-					: __( 'Activate PageSpeed Insights.', 'google-site-kit' )
-				}
-				description={ __( 'Google PageSpeed Insights gives you metrics about performance, accessibility, SEO and PWA.', 'google-site-kit' ) }
-				ctaLink={ '#' }
-				ctaLabel={ ! connected && active
-					? __( 'Complete activation', 'google-site-kit' )
-					: __( 'Activate PageSpeed Insights', 'google-site-kit' )
-				}
-				onClick={ handleSetUpClick }
-			/>
+			{
+				( ! connected && active )
```

Comment:
Let's simplify this code a bit, avoiding the `if else` in JSX: We can have a single if clause `! active` that returns the `ActivateModuleCTA` with an early return. And then afterwards (due to the logic before this will mean the module is active but not connected) return the `CompleteModuleActivationCTA`.

Reply:
@felixarntz Not sure if we can do an early return here since the components are within 
```
<div className="mdc-layout-grid__cell mdc-layout-grid__cell--span-12">
```
Let me know if you want the logic to be extracted to a function where we can do an early return instead?